src/wallet/rpcwallet.cpp

```diff
@@ -29,7 +29,8 @@
 
 CWallet *GetWalletForJSONRPCRequest(const JSONRPCRequest& request)
 {
-    return pwalletMain;
+    // TODO: Some way to access secondary wallets
```

Comment:
I throwing an RPC error like `RPC_METHOD_NOT_FOUND` might be better than returning `NULL`.